dev/AppLifecycle/FileActivatedEventArgs.h

```diff
@@ -10,9 +10,9 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
     using namespace winrt::Windows::ApplicationModel::Activation;
     using namespace winrt::Windows::Storage;
 
-    static PCWSTR c_fileContractId = L"Windows.File";
+    constexpr PCWSTR c_fileContractId = L"Windows.File";
 
```

Comment:
Do we need to say `constexpr inline` to avoid ODR violations?

> A static member variable (but not a namespace-scope variable) declared constexpr is implicitly an inline variable.

[https://en.cppreference.com/w/cpp/language/inline](https://en.cppreference.com/w/cpp/language/inline)